docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -527,6 +529,25 @@ The following things are validated:
 
 Any errors found will be logged with error level to ``stderr``.
 
+scrub
+^^^^^
+
+Rewrites the sstable, skipping or fixing corrupt parts. Not all kind of corruptions can be skipped or fixed by scrub.
+Currently this is limited to ordering issues on the partition, row or mutation-fragment level, see `sstable content <scylla-sstable-sstable-content_>`_ for more details.
+
+Scrub has several modes:
+
+* **abort** - abort the scrub as soon as any error is found (recognized or not). This mode is not all that useful (**validate** mode is strictly an improvement over **abort** mode), it is included for completeness' sake.
+* **skip** - skip over any corruptions found, thus omitting them from the output. Note that skip can end up omitting more than was strictly necessary, but it is guaranteed that it will skip all detectable corruptions.
+* **segregate** - fix partition/row/mutation-fragment out-of-order errors, by segregating the output into as many sstables as required, such that the content of each output sstable is properly ordered.
+* **validate** - validate the content of the sstable, reporting any corruptions found. Writes no output sstables. This is the same as the `validate operation <scylla-sstable-validate-operation_>`_ and it is recommended that the latter is used instead.
+
+Output sstables are written to the directory specified via ``--output-directory``. They will be written with the ``BIG`` format and the highest supported sstable format, with generations choosen by scylla-sstable. Generations are chosen such
+that they are unique between the sstables written by the current scrub.
+
+The output directory is expected to be empty, if it isn't, scylla-sstable will abort the scrub. This can be overriden by the ``--unsafe-accept-nonempty-output-dir`` command line flag.
```

Comment:
```suggestion
The output directory must be empty; otherwise, scylla-sstable will abort scrub. You can allow writing to a non-empty directory by setting the ``--unsafe-accept-nonempty-output-dir`` command line flag.
```